uber/site_sections/summary.py

```diff
@@ -317,17 +318,14 @@ def shirt_manufacturing_counts(self, session):
 
         for attendee in session.all_attendees():
             shirt_label = attendee.shirt_label or 'size unknown'
-
-            if attendee.gets_staff_shirt:
-                counts['staff'][label(shirt_label)] += c.SHIRTS_PER_STAFFER
-
-            counts['swag'][label(shirt_label)] += attendee.num_swag_shirts_owed
+            counts['staff'][label(shirt_label)] += attendee.num_staff_shirts_owed
+            counts['event'][label(shirt_label)] += attendee.num_event_shirts_owed
```

Comment:
In order to test these shirt count reports, I took a snapshot of our production database and loaded it onto my machine and then did some manual database queries to compute a few of the numbers by hand to make sure that they matched what's being spit out in these reports.  This wasn't exhaustive, but it was enough that I feel confident in these numbers.